src/dot-components/Cart/cart.css

```diff
@@ -0,0 +1,105 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,300;0,400;0,500;0,900;1,300&display=swap');
+
+body{
+    font-family: "Montserrat", sans-serif;
+}
+
+.cart-container{
+    display: flex;
+    margin-right: 50px;
+
```

Comment:
extra spaces at somplaces, can use some plugin for formatting

Reply:
Beautify and such, right? Will do that

Reply:
prettify would work fine.